Makefile

```diff
@@ -65,19 +60,18 @@ lint-go:
 
 lint: ensure-submodule-initialized lint-cpp lint-go
 
-setup: ensure-submodule-initialized set-pkg-config-openssl
+setup: ensure-submodule-initialized
 ifeq ("Darwin", "$(shell uname -s)")
 	cd viam-cartographer/scripts && ./setup_cartographer_macos.sh
 else
 	cd viam-cartographer/scripts && ./setup_cartographer_linux.sh
```

Comment:
Thanks James. As far as I understand this would have never worked on Linux then, even before I made these changes? If so I may wait until @EshaMaharishi is back to discuss and reprioritize, since this was supposed to be a one-liner to get this to work on both Intel and Apple chip Macs and snowballed into a larger refactor & fix of multiple issues in the setup scripts and Makefile.

Reply:
Yes, it was certainly broken in many cases beforehand as well. I was going on the title that said "...correctly on all platforms" and assumed this was intended to fix up ALL platforms. That said, I'm fine if your team feels that's out of scope here, and this is really just "Find OpenSSL on both intel/arm macs"

Reply:
That makes sense, the PR title was very ambitious 🙂 I'll change the PR & ticket names and file a ticket to handle building on Linux, address your other comments, and then re-request. Thank you!